app/views/welcome/index.html.erb

```diff
@@ -1 +1 @@
-<h1>Welcome!</h1>
+<h1>Welcome to MONSTER SHOP!</h1>
```

Comment:
Little changes like this make big impacts on how professional it looks. well done